client/src/app/splash-page/splash-page.component.html

```diff
@@ -3,9 +3,7 @@
   <div class="spash_copy">
     <p></p>
   </div>
-  <button class="btn btn-default" (click)="login()">Login</button>
```

Comment:
I believe this section is the most recent version of the splash page 🙈 Maybe you should add these lines of code back (the red ones) and delete the green ones! Sorry about that! I think it may have been caused by a merge conflict!

Reply:
Hey Maria! I am working on this right now. So, I am changing it to line 6 - 9, correct?

Reply:
Sorry, this is super late of me!!! But based on this above code, you are all good!